tests/hil/platform/zephyr/prj.conf

```diff
@@ -20,7 +19,6 @@ CONFIG_IMG_MANAGER=y
 CONFIG_NVS=y
 CONFIG_SETTINGS=y
 CONFIG_SETTINGS_RUNTIME=y
-CONFIG_GOLIOTH_SAMPLE_SETTINGS=y
 CONFIG_GOLIOTH_SAMPLE_SETTINGS_AUTOLOAD=y
 CONFIG_GOLIOTH_SAMPLE_SETTINGS_SHELL=y
```

Comment:
GitHub won't take this in "suggestion" format, but I think lines 24-25 can be deleted as they'll be automatically selected:

```
CONFIG_GOLIOTH_SAMPLE_SETTINGS_AUTOLOAD=y
CONFIG_GOLIOTH_SAMPLE_SETTINGS_SHELL=y
```